.hemtt/launch.toml

```diff
@@ -54,4 +54,47 @@ workshop = [
 extends = "cup"
 workshop = [
     "1369691841", # Community Factions Project
-]
\ No newline at end of file
+]
+
+[wardrobe]
+extends = "default"
+mission = "Wardrobe.VR"
+workshop = [
+    "3499977893", # Advanced Developer Tools
+    "1779063631", # ZEN
+#    "623475643",  # eden enhanced
+]
+
+[wardrobe_gm]
+extends = "wardrobe"
+dlc = ["gm"]
+
+[wardrobe_ws]
+extends = "wardrobe"
+dlc = ["Western Sahara"]
+
+[wardrobe_rf]
+extends = "wardrobe"
+dlc = ["Reaction Forces"]
+
```

Comment:
We don't need a separate wardrobe launch profile for each mod and dlcs

You just do `hemtt launch rf wardrobe` to combine them